Mathlib/GroupTheory/Perm/Finite.lean

```diff
@@ -251,3 +251,12 @@ theorem support_pow_coprime {σ : Perm α} {n : ℕ} (h : Nat.Coprime n (orderOf
 end Fintype
 
 end Equiv.Perm
+
+namespace Nat
+
+theorem card_perm {α : Type*} [Finite α] : Nat.card (Equiv.Perm α) = (Nat.card α)! := by
+  classical
+  have := Fintype.ofFinite α
+  rw [Nat.card_eq_fintype_card, Nat.card_eq_fintype_card, Fintype.card_perm]
+
+end Nat
```

Comment:
It looks to me like this doesn't belong in a GroupTheory file, and instead could be in `Mathlib.SetTheory.Cardinal.Finite`.

Reply:
I put it there.